libbindgen/src/ir/function.rs

```diff
@@ -147,6 +147,7 @@ impl FunctionSig {
         };
         let mut args: Vec<_> = match cursor.kind() {
             CXCursor_FunctionDecl |
+            CXCursor_Constructor |
             CXCursor_CXXMethod => {
```

Comment:
Ah -- are we not binding destructors yet? That's surprising I really thought we were.